osp/serializers/form_feedback.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import serializers
+
+from osp.models import FormFeedback
+from osp.serializers.form import FormSerializer
+from osp.serializers.user import UserSerializer
+from osp.serializers.answer import AnswerReadSerializer
+
+class FormFeedbackReadSerializer(serializers.ModelSerializer):
+
+    form = FormSerializer(read_only=True)
+    user = UserSerializer(read_only=True)
+    answers = AnswerReadSerializer(read_only=True, many=True)
```

Comment:
https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield
Use this
I guess this will be done in the previous PR and you just need to rebase it